### PR TITLE
wip - test(e2e): enable RBAC tests RHIDP-5208

### PR DESCRIPTION
**Pull request by nilgaar**
## Description

Please explain the changes you made here.

## Which issue(s) does this PR fix

- Fixes [RHIDP-4264](https://issues.redhat.com//browse/RHIDP-4264)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+});
+
+base.describe.serial("Test RBAC plugin as an admin user", () => {
+  base.beforeAll(async ({ browser }, testInfo) => {
+    myPage = (await setupBrowser(browser, testInfo)).page;
+    await new Common(myPage).loginAsGithubUser();
+  });
+
+  base.beforeEach(
+    async () => await new Common(myPage).checkAndClickOnGHloginPopup(),
```
Comment (subhashkhileri) [inline e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts:50]: checkAndClickOnGHloginPopup isn't required unless the plugin requires github user login

File: e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+      await uiHelper.clickButton("Next");
+      await uiHelper.clickButton("Save");
+      await uiHelper.verifyText(
+        `Role ${testId.composedRole} updated successfully`,
+      );
+    },
+  );
+
+  base.fixme(
```
Comment (subhashkhileri) [inline e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts:232]: can you please comment whats the issue and why this is fixme.

File: e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts (hunk 1 above)
Comment (nilgaar) [reply e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts:50]: Isn't this the case? We are doing Github Login in the set

Comment (nilgaar) [reply e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts:50]: * or do you mean that github login is not required for these tests?

Comment (subhashkhileri) [reply e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts:50]: checkAndClickOnGHloginPopup was meant for plugins that require additional login apart from backstage login even though you logged in with github it still asks. like github-actions, issues, pr/mr are few plugins that require this.

Comment (nilgaar) [reply e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts:50]: I think this is also necessary in some other cases, like happened here:
https://redhat-internal.slack.com/archives/C04CUSD4JSG/p1730884919450929?thread_ts=1730730693.542309&cid=C04CUSD4JSG
But I am not quite sure, I'll try to remove it

**Pull request closed without merging**